client/patients/get-funded.js

```diff
@@ -33,7 +16,16 @@ angular.module('eir.getFunded', [])
 
     // only submit if form is valid
     if(form.$valid) {
-      fileUpload.uploadFileToUrl($scope.patient);
+      fileUpload.uploadFileToUrl($scope.patient)
+        .then(function (res, err){
+          // redirect patient to their new page
+          $location.path('/donate/' + res.data.insertId);
+        })
+        .catch(function (err, res){
```

Comment:
@benjaminhoffman here I am catching any and all errors while trying to submit a new patient. these would be
- duplicate username
- invalid email
- mysql server not online
- etc

I still have to handle these errors and this is what I am doing now. For the most part the one error that will likely come back will be duplicate username, which I then need to display a box next to the username with some text like 'already taken, choose new username'